notebooks/240-named-entity-recognition/240-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>can we also provide description about export=True and saving model using save_pretrained?</p><p>I think it can be important to compare performance quantized and original model in notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
added